### PR TITLE
Ic pomodoro restmapping hotfix

### PR DESCRIPTION
**Pull request by ivanCosic**

### CODE REVIEW

File: service/src/test/java/eu/execom/labs/pomodorospringreact/service/TeamServiceTest.java [hunk 1]
```diff
@@ -369,4 +370,4 @@
 //        return teamList;
 //    }
 //}
-//
+//
```
Comment (dejanjesic) [inline service/src/test/java/eu/execom/labs/pomodorospringreact/service/TeamServiceTest.java]: Missing empty line at the end of file

File: web/src/main/java/eu/execom/labs/pomodorospringreact/controller/PomodoroController.java [hunk 2]
```diff
@@ -13,36 +13,41 @@
...
+@RequestMapping(value = "/api/v1/pomodoros", headers = "Accept=*/*", produces = "application/json")
 public class PomodoroController {
 
     @Autowired
     private PomodoroService pomodoroService;
 
-    @RequestMapping(method = RequestMethod.POST, value = "user/{id}/start")
+    @RequestMapping(method = RequestMethod.GET, value = "/{id}")
+    public ResponseEntity<PomodoroDto> getPomodoro(@PathVariable("id") long pomodoroId) {
+        return new ResponseEntity<PomodoroDto>(pomodoroService.find(pomodoroId), HttpStatus.OK);
```
Comment (dejanjesic) [inline web/src/main/java/eu/execom/labs/pomodorospringreact/controller/PomodoroController.java]: You don't need to explicitly fill diamond operator, compiler will recognize it.

File: web/src/test/java/eu/execom/labs/pomodorospringreact/TeamControllerTest.java [hunk 3]
```diff
@@ -292,4 +292,4 @@
 //
 //        return userDtoList;
 //    }
-//}
+//}
```
Comment (dejanjesic) [inline web/src/test/java/eu/execom/labs/pomodorospringreact/TeamControllerTest.java]: As above

File: web/src/main/java/eu/execom/labs/pomodorospringreact/controller/PomodoroController.java (hunk 2 above)
Comment (ivanCosic) [reply web/src/main/java/eu/execom/labs/pomodorospringreact/controller/PomodoroController.java]: It's filled like that in the whole PomodoroController class, i didn't want to refactor that at the moment, and I didn't want to break the consistency only for this one.

File: service/src/test/java/eu/execom/labs/pomodorospringreact/service/PomodoroServiceParameterizedExceptionPathTest.java [hunk 4]
```diff
@@ -1,132 +1,132 @@
...
+//
+//        //when - then throw DurationOutOfBoundariesException
+//        try {
+//            pomodoroService.start(userId, customDuration, customShortPauseDuration, customLongPauseDuration);
+//        } catch (DurationOutOfBoundariesException e) {
+//            assertEquals(expectedExceptionMessage, e.getMessage());
+//            throw e;
+//        }
+//    }
+//}
```
Comment (dejanjesic) [inline service/src/test/java/eu/execom/labs/pomodorospringreact/service/PomodoroServiceParameterizedExceptionPathTest.java]: missing line

File: service/src/test/java/eu/execom/labs/pomodorospringreact/service/PomodoroServiceParameterizedHappyPathTest.java [hunk 5]
```diff
@@ -1,165 +1,165 @@
...
+//        assertEquals(userId, pomodoroDto.getUserId());
+//        assertEquals(PomodoroState.RUNNING, pomodoroDto.getState());
+//        assertEquals(expectedDuration, pomodoroDto.getDuration());
+//        assertEquals(expectedDuration, Minutes.minutesBetween(
+//                new DateTime(pomodoroDto.getStartTime()),
+//                new DateTime(pomodoroDto.getEndTime())
+//        ).getMinutes());
+//        assertEquals(expectedRestDuration, pomodoroDto.getProposedRest());
+//    }
+//}
```
Comment (dejanjesic) [inline service/src/test/java/eu/execom/labs/pomodorospringreact/service/PomodoroServiceParameterizedHappyPathTest.java]: as above

File: web/src/main/java/eu/execom/labs/pomodorospringreact/controller/TeamController.java [hunk 6]
```diff
@@ -29,22 +29,22 @@
         return new ResponseEntity<List<TeamDto>>(teamService.getAllTeams(), HttpStatus.OK);
     }
 
-    @RequestMapping(method = RequestMethod.GET, value = "{teamId}")
+    @RequestMapping(method = RequestMethod.POST)
+    public ResponseEntity<TeamDto> createTeam(@RequestBody NewTeamDto newTeamDto) {
+        return new ResponseEntity<TeamDto>(teamService.createTeam(newTeamDto), HttpStatus.CREATED);
```
Comment (dejanjesic) [inline web/src/main/java/eu/execom/labs/pomodorospringreact/controller/TeamController.java]: as above

**Pull request merged**